ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh

```diff
@@ -10,9 +10,6 @@ if [ "${TEST_TYPE}" == "conformance-serial" ]; then
 "[sig-imageregistry][Serial][Suite:openshift/registry/serial] Image signature workflow can push a signed image to openshift registry and verify it [Suite:openshift/conformance/serial]"
 EOF
 fi
-if [ "${TEST_TYPE}" != "conformance-parallel" ]; then
```

Comment:
It will be great if we can add more information to the PR description about the change we are making, this sounds like we are now supporting more suites than just `conformance-parallel`.

need to fix the message here, which is not true anymore.

https://github.com/openshift/release/blob/4c663c1225d89980bf813824c0c769a1cba678ce/ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh#L288